app/tests/src/ingestion/test_pdf_postprocess.py

```diff
@@ -41,28 +45,37 @@ def test_add_markdown():
         ),
     ]
 
-    result = add_markdown(enriched_texts)
 
-    assert result == [
+@pytest.fixture
```

Comment:
What's the reasoning for turning an expected result into a test fixture? https://blog.machinet.net/post/what-is-a-test-fixture-a-comprehensive-guide